packages/adapter-prisma/lib/adapter-prisma.js

```diff
@@ -0,0 +1,611 @@
+const fs = require('fs');
+const path = require('path');
+const { execSync } = require('child_process');
+const cuid = require('cuid');
+const { getGenerators, formatSchema } = require('@prisma/sdk');
+const { BaseKeystoneAdapter, BaseListAdapter, BaseFieldAdapter } = require('@keystonejs/keystone');
+const { defaultObj, mapKeys, identity, flatten } = require('@keystonejs/utils');
+
+class PrismaAdapter extends BaseKeystoneAdapter {
+  constructor() {
+    super(...arguments);
+    this.name = 'prisma';
+
+    this.getPrismaPath = this.config.getPrismaPath || (() => '.prisma');
+    this.getDbSchemaName = this.config.getDbSchemaName || (() => 'public');
+    this.enableLogging = this.config.enableLogging || false;
+    this.url = this.config.url || process.env.DATABASE_URL;
+  }
+
+  async _connect({ rels }) {
+    await this._generateClient(rels);
+    const { PrismaClient } = require(this.clientPath);
+    this.prisma = new PrismaClient({
+      log: this.enableLogging && ['query'],
+      datasources: { postgresql: { url: this._url() } },
+    });
```

Comment:
Shouldn't we explicitly call and await .$connect() here so we're ready right away rather than on the first call to prisma?

Reply:
Yep, we can do that 👍